andela/ah-backend-sparta
# #164798206 Users should rate articles

#### What does this PR do?
This PR enables a user to rate an article, between the range of 0-5

#### Description of Task to be completed?
A user can leave a rating on an article but not more than once

#### How should this be manually tested?
run `git fetch origin users-should-rate-articles-164798206`
checkout `git checkout users-should-rate-articles-164798206`
enter shell `pipenv shell`
run migrations `python manage.py migrate`
runserver `python manage.py runserver`

#### Any background context you want to provide?
To rate an article go to the endpoint `articles/<slug>/rate` and send rating as a JSON object
`{"ratings": <0 - 5>}`

#### What are the relevant pivotal tracker stories?
[#164798206](https://www.pivotaltracker.com/story/show/164798206)

#### Postman collection
[![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/f17327eecaa858764bb6)
